src/parsec_service/install_parsec_linux.md

```diff
@@ -6,13 +6,14 @@ up correctly so that Parsec is installed respecting the operational mitigations
 model](../parsec_security/parsec_threat_model/threat_model.md).
 
 If your Linux system uses systemd to manage daemons, you can follow these steps. `$DESIRED_FEATURES`
-can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-provider`, and
-`tpm-provider`. Choose the providers you want to install depending on what is available on the
-platform.
+can be a space or comma-separated subset of `XXX-provider` and `XXX-authenticator` features. Choose
```

Comment:
Are we sure `XXX` is the best placeholder here? 😁 Google (and other web filtering software) might flag the book as naughty just based on that

Reply:
Will change to `*`